README.md

```diff
@@ -17,46 +17,177 @@ Or directly:
 
 The bitpay client creates a cryptographically secure connection to your server by pairing an API code with keys generated by the library. The client can be initialized with pre-existing keys passed in as a pem file, or paired if initialized with a pem file and a tokens hash. Examples can be found in the cucumber step helpers.
 
-## Basic Usage
+## Command-line Interface (CLI)
+
+BitPay also provides a command-line interface (CLI) which can be used to generate client-side cryptographic credentials and for client pairing with the BitPay service.  Generally speaking, this is the easiest way to pair your client.  The library must be installed as a local gem.
+
+    gem install bitpay-client
+    
+Source code and documentation for the Ruby CLI can be found at https://github.com/bitpay/ruby-cli
+
+## Client Setup
 
 ### Pairing with Bitpay.com
 
-To pair with bitpay.com you need to have an approved merchant account.  
+Most calls to the BitPay REST API require that your client is paired with the bitpay.com server.  To pair with bitpay.com you need to have an approved merchant account.
+
+Your client can be paired via the `pos` (point-of-sale) or `merchant` facade (or both).  The `pos` facade allows for invoices to be created.  The `merchant` facade has broader privileges to view all invoices, bills, and ledger entries, as well as to issue refunds.  Consider the level of access required when you pair your client.
+
+Pairing can be performed via two separate methods:
+
+#### To initiate pairing from the server
+_Suited for POS clients where pre-authorized tokens are desirable_
+
 1. Login to your account  
 2. Navigate to bitpay.com/api-tokens (Dashboard > My Account > API Tokens)  
 3. Create a new token and copy the pairing code.  
+4. Use the bitpay command line tool to pair with bitpay.com `bitpay pair <pairing_code>`
```

Comment:
We should not assume the installation of a downstream gem in the instructions for using this gem.